src/components/nav-drawer/styles/nav-drawer.base.scss

```diff
@@ -77,6 +77,15 @@
     }
 }
 
+:host([position='relative']) {
+    [part='base'] {
+        position: relative;
+        width: fit-content;
+        height: 100vh;
```

Comment:
The 100% height from the base styling is just fine, so no need for this. The drawer can actually sit under a nav bar commonly and 100vh will mess with that, so let the surrounding layout control the height.

For the stories demo to show correctly you can simply expand the main container like so:
https://github.com/IgniteUI/igniteui-webcomponents/commit/200afb16864330f9c0dfc4919a22ec25484a3763#diff-21d8dd568fac18b25ddeb66244307d0251e27ea735ae64b36ff2ec202297507dR202

Reply:
Do you suggest adding a 'mini' property or just adding control in the story?

Reply:
That's separate, the link should be directly to the line that sets the story container height and you've added it so it's good now